### PR TITLE
feat(storybook): added withStore decorator for Storybook

### PR DESCRIPTION
**Pull request by mukiienko**

### CODE REVIEW

File: stories/Autosuggest.js [hunk 1]
```diff
@@ -52,85 +65,79 @@ storiesOf('Organisms|Autosuggest', module)
...
+        return (
+            <Autosuggest
+                selectedSuggestions={
+                    boolean('Add selectedSuggestions', true)
+                        ? store.get('selectedSuggestions')
+                        : undefined
+                }
+                selectedPlaceholder={
+                    boolean('Add selectionPlaceholder', true) ? getSelectedPlaceholder() : undefined
+                }
```
Comment (eszthoff) [inline stories/Autosuggest.js]: Maybe we can add an icon with this same trick to render or not depending on the knob.

Comment (mukiienko) [reply stories/Autosuggest.js]: are you mean something like this?
`
iconNode={
                        boolean('Add iconNode', true)
                            ? <IconMatch />
                            : undefined
                    }
`

Comment (eszthoff) [reply stories/Autosuggest.js]: yes. Then all props are covered

File: stories/Autosuggest.js [hunk 2]
```diff
@@ -51,86 +58,81 @@ storiesOf('Organisms|Autosuggest', module)
...
+            store.set({ inputValue: '' });
         };
 
-        Implementation.displayName = 'Implementation';
+        const onClearAllSelected = () => {
+            console.log('onClearAllSelected was called');
+            store.set({ selectedSuggestions: [] });
+        };
 
-        return <Implementation style={{ width: '650px' }} />;
```
Comment (eszthoff) [inline stories/Autosuggest.js]: It looks better in storybook if we keep the style. you can add it directly to Autosuggest

Comment (mukiienko) [reply stories/Autosuggest.js]: Are you mean to add this style just in the last story  **Example implementation** or to all **Autosuggest**?

**Pull request merged**